Field Service Management/documents/public/css/servicedesk.css

```diff
@@ -6,6 +6,7 @@
 	width:150px;
 	border-radius: 5px;
 	transition: .4s;
+    color: white !important;
```

Comment:
This one line is still off by a tab length, otherwise everything here looks good